README.md

```diff
@@ -36,3 +36,10 @@ The color pallete has 7 different colors for selection **( Red , Yello , Green ,
 <img src="https://github.com/anshumyname/drawing-sheet/blob/master/img.jpg">
 
 
+# drawing-sheet
+
```

Comment:
Add this point in the beginning of the readme

Reply:
Okk